go/vt/dbconfigs/dbconfigs.go

```diff
@@ -73,10 +73,11 @@ const (
 	Dba      = "dba"
 	Filtered = "filtered"
 	Repl     = "repl"
+	Backup   = "backup"
 )
 
 // All can be used to register all flags: RegisterFlags(All...)
-var All = []string{App, AppDebug, AllPrivs, Dba, Filtered, Repl}
+var All = []string{App, AppDebug, AllPrivs, Dba, Filtered, Repl, Backup}
```

Comment:
I'm thinking that we should not go down the route of adding one more dbconfig. It seems to me that this adds some incidental complexity:

1. We need to modify `Backup`, and `RestoreBackup` to pass dbconfigs all the way down to the engine. This modifies all engines and right now only xtrabackup needs it. 
2. At the moment we are adding all this wiring, only to get the username for the backup user. 

I would propose a simpler approach where we add a flag in the xtrabackup engine to specify the username. In the future when we figure out how to solve the issue of passing the password, we can iterate and either add one more flag, or revisit this part.

Reply:
I'm fine with that. @sougou opinions?

Reply:
replaced `backup` user with a single flag called `xtrabackup_user` for this iteration.